postgresql-11.5/src/bin/psql/commandbteq.c

```diff
@@ -147,10 +150,11 @@ HandleDotCmds(BteqScanState scan_state,
 
         conditional_stack_push(cstack, IFSTATE_IGNORED);
         while ((arg = bteq_scan_dot_option(scan_state,
-                                             OT_BTEQ_NORMAL, NULL, false)))
+                                             OT_BTEQ_NORMAL, NULL, true)))
         {
             if (active_branch)
-                psql_error("\\%s: extra argument \"%s\" ignored\n", cmd, arg);
```

Comment:
Test the following case:
`.set width 20 extra aaaaa;`
I believe it will not work properly. Need to change the mechanism of getting extra text.

Reply:
Also, I think psql_error function prints error in the format similar to this:
`program name:source file name:line number: error message.`
We do not need this in BTEQ mode, let's simply use `printf` function.